README.md

```diff
@@ -75,10 +75,9 @@ After you're finished please remove all the comments and instructions!
 - **[Styling css with colors, borders, paddings, margins & sizes, etc]**
 _ **[React, Redux & javascript functions, modules, ES6 syntax, variables, React testing, etc]**
```

Comment:
- [ ] [OPTIONAL]
I think the ReadMe has less spacing which makes it look cluttered. Can we improve upon this by ensuring consistent spacing as seen in this ReadMe [template](https://github.com/microverseinc/readme-template)?